File: app/controllers/Proconsorcio/LandingPage.java
package controllers.Proconsorcio;

import play.mvc.Controller;
import play.mvc.Result;
import securesocial.core.java.SecureSocial;


/**
 * Created by claytonsantosdasilva on 08/06/14.
 */
public class LandingPage extends Controller {

	@SecureSocial.UserAwareAction
    public static Result index()
    {
		dao.UserIdentity user = (dao.UserIdentity) ctx().args.get(SecureSocial.USER_KEY);
        
        
    	if (user != null){
    		 
    		return redirect("/home");
    			
    		 
    	}else
    	{
    		 return ok(views.html.LandingPage.main.render());
    		 
    		 
    	}
    
    }


}

File: app/controllers/Proconsorcio/LandingPage.scala
package controllers.Proconsorcio

import controllers.xDevController
import dao.UserIdentity
import play.api.mvc._
import securesocial.core.java.SecureSocial


/**
 * Created by claytonsantosdasilva on 10/07/14.
 *
 * Controller responsável pelo processamento da landingPage
 *
 */
object LandingPage extends xDevController{
  def Home = UserAwareAction { implicit request =>
    if(!_user.isDefined){Ok(views.html.LandingPage.main())}else{Redirect("/home")}
  }

}
